spec/models/user_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  # tests go here
+  describe 'validations' do
+    it 'is invalid without a name' do
+      user = User.new(name: nil)
+      user.valid?
+      expect(user.errors).to have_key(:name)
+    end
+
+    it 'is invalid with a posts_counter less than 0' do
+      user = User.new(posts_counter: -1)
+      user.valid?
+      expect(user.errors).to have_key(:posts_counter)
+    end
+  end

```

Comment:
- [ ] Kindly add some positive test cases also. For ex
- Create one user and check its validity & its attributes

Reply:
Thank you for the suggestion. I will make the change